plugins/woocommerce/src/Internal/RestApi/Routes/V4/Orders/Schema/OrderCouponSchema.php

```
@@ -95,28 +95,17 @@ public function get_item_schema_properties(): array {
 	 */
 	public function get_item_response( $order_item, WP_REST_Request $request, array $include_fields = array() ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$include_fields'. (undefined)

(UnusedFormalParameter)